src/pcapi/core/search/backends/appsearch.py

```diff
@@ -28,6 +28,7 @@
 
 
 SCHEMA = {
+    "appLabel": "text",
```

Comment:
les champs sont en snake_case :

```suggestion
    "app_label": "text",
```

De plus, "app_label" ne me parle pas trop. Si c'est la sous-catégorie de l'offre, est-ce que subcategory_label ne serait pas un nom plus parlant ?

Reply:
:+1: